### PR TITLE
Mordred

### PR DESCRIPTION
**Pull request by simonbray**
Molecular descriptor tool @khanteymoori can use for feature extraction.

### CODE REVIEW

File: chemicaltoolbox/mordred/mordred_descriptors.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <param name="use_3d" value="False" />
+            <output name="output" ftype='tabular' file="8smi.tab" />
+        </test>
+    </tests>
+
+    <help><![CDATA[
+Calculates up to 1825 molecular descriptors using the Mordred package. A list of all
+descriptors is located here_.
+
+.. _here: https://static-content.springer.com/esm/art%3A10.1186%2Fs13321-018-0258-y/MediaObjects/13321_2018_258_MOESM3_ESM.xlsx
```
Comment (bgruening) [inline chemicaltoolbox/mordred/mordred_descriptors.xml]: Can we dump this into github somewhere and link to it?

File: chemicaltoolbox/mordred/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- Computational chemistry
+description: Mordred is a molecular descriptor calculator.
+long_description: |
+  Mordred is a descriptor-calculation software application that can calculate more than 1800 two- and three-dimensional descriptors.
+  
+  Home page: https://github.com/mordred-descriptor/mordred
+  License: BSD
+
+homepage_url: https://github.com/mordred-descriptor/mordred
+auto_tool_repositories:
```
Comment (bgruening) [inline chemicaltoolbox/mordred/.shed.yml]: are you planning to add more tools in this repo? If not these auto_tool_repositories are not needed as you only have one tool

File: chemicaltoolbox/mordred/mordred_descriptors.xml (hunk 1 above)
Comment (simonbray) [reply chemicaltoolbox/mordred/mordred_descriptors.xml]: this okay? https://github.com/simonbray/mordred-descriptors

**Comment by bgruening** [comment]
Perfect!

**Comment by bgruening** [comment]
Tests are failing as it seems :(

**Comment by bgruening** [comment]
Sieht gut aus!

**Pull request merged**